skrub/datasets/_utils.py

```diff
@@ -54,3 +81,155 @@ def get_data_dir(name=None, data_home=None):
     if name is not None:
         data_dir = data_dir / name
     return data_dir
+
+
+def load_simple_dataset(dataset_name, data_home=None):
+    """TODO
+    """
+    bunch = _load_dataset_files(dataset_name, data_home)
+    bunch["X"] = bunch.pop(dataset_name)
+    if (target := bunch.metadata.get("target", None)) is not None:
+        bunch["y"] = bunch["X"][target]
+        bunch["X"] = bunch["X"].drop(columns=target)
+    return bunch
+
+
+def _load_dataset_files(dataset_name, data_home):
+    """
+    TODO
+    skrub_data/
+        fraud/
+            fraud.tar.gz
+            fraud/
+                baskets.csv
+                products.csv
+                metadata.json
+    """
+    data_home = get_data_home(data_home)
+    dataset_dir = data_home / dataset_name
+    datafiles_dir = dataset_dir / dataset_name
+    datafiles_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
I think we could create the `datafiles_dir` only once we have successfully extracted the archive's content, that way we can simply check its existence (see below)